orangecontrib/educational/widgets/owkmeans.py

```diff
@@ -373,17 +354,17 @@ def replot(self):
         if self.data is None or not self.attr_x or not self.attr_y:
             return
 
-        if self.k_means.centroids_moved:
-            # when centroids moved during step
-            self.scatter.update_series(0, self.k_means.centroids)
-
-            if self.lines_to_centroids:
-                for i, c in enumerate(self.k_means.centroids):
-                    self.scatter.update_series(1 + i, list(chain.from_iterable(
-                        ([p[0], p[1]], [c[0], c[1]])
-                        for p in self.k_means.centroids_belonging_points[i])))
-        else:
+        km = self.k_means
+        if not km.centroids_moved:
```

Comment:
We aim to not exceed 80 columns. (I work on laptop and like to see two files in parallel, so I'm rather strict about this rule, but I haven't enforced it here.)

This is why in this case I would handle the trivial case (centroids haven't moved) first and avoid the `else` part, hence saving four characters.